### ISSUE TITLE
nats-top columns becomes offset if IPv6 is used

### ISSUE THREAD

**Issue by sixlettervariables** [nats-io/nats-top#35]
If IPv6 is used the columns in nats-top become offset. Not impacting functionality, just legibility.

```
NATS server version 0.8.0 (uptime: 26d2h18m19s)
Server:
  Load: CPU:  0.0%  Memory: 9.4M  Slow Consumers: 0
  In:   Msgs: 3.0M  Bytes: 1.7G  Msgs/Sec: 5.0  Bytes/Sec: 407
  Out:  Msgs: 2.9M  Bytes: 1.7G  Msgs/Sec: 4.0  Bytes/Sec: 405

Connections: 19
  HOST                 CID      NAME            SUBS    PENDING     MSGS_TO     MSGS_FROM   BYTES_TO    BYTES_FROM  LANG     VERSION  UPTIME   LAST ACTIVITY
  fe80::1ca1:378d:181f:1d32:58989 360                      0       0           0           117         0           24.2K       .NET     0.0.1    13d2h23m13s  2016-06-01 13:47:49.3332254 -0400 EDT
  fe80::1ca1:378d:181f:1d32:58999 362                      0       0           0           13          0           4.6K        .NET     0.0.1    13d2h22m44s  2016-06-01 13:14:38.6280105 -0400 EDT
  fe80::1ca1:378d:181f:1d32:50790 402                      31      0           370.3K      431.4K      421.6M      421.7M      .NET     0.0.1    10d20h43m52s  2016-06-14 14:18:56.5726337 -0400 EDT
  fe80::1ca1:378d:181f:1d32:50799 403                      0       0           0           144         0           42.8K       .NET     0.0.1    10d20h43m32s  2016-06-14 09:31:06.6342725 -0400 EDT
  fe80::1ca1:378d:181f:1d32:50804 404                      0       0           0           157         0           46.7K       .NET     0.0.1    10d20h42m44s  2016-06-14 13:31:05.9336501 -0400 EDT
  fe80::1ca1:378d:181f:1d32:55543 437                      0       0           0           59          0           16.8K       .NET     0.0.1    4d3h43m52s  2016-06-14 13:30:32.0765693 -0400 EDT
  fe80::1ca1:378d:181f:1d32:55610 438                      32      0           188.2K      211.6K      85.5M       85.6M       .NET     0.0.1    4d3h33m14s  2016-06-14 14:18:52.6675596 -0400 EDT
  fe80::1ca1:378d:181f:1d32:55621 439                      0       0           0           50          0           14.2K       .NET     0.0.1    4d3h33m2s  2016-06-14 11:30:30.6180111 -0400 EDT
  fe80::1ca1:378d:181f:1d32:63946 442                      34      0           140.0K      163.1K      159.4M      159.4M      .NET     0.0.1    4d2h34m56s  2016-06-14 14:18:55.3795956 -0400 EDT
  fe80::1ca1:378d:181f:1d32:63957 443                      0       0           0           53          0           15.6K       .NET     0.0.1    4d2h34m38s  2016-06-14 09:31:06.5612718 -0400 EDT
  fe80::1ca1:378d:181f:1d32:63962 444                      0       0           0           52          0           14.7K       .NET     0.0.1    4d2h33m57s  2016-06-14 13:31:05.859648 -0400 EDT
  fe80::1ca1:378d:181f:1d32:61690 449                      0       0           0           6           0           1.8K        .NET     0.0.1    4d0h47m46s  2016-06-10 14:12:12.5593914 -0400 EDT
  fe80::1ca1:378d:181f:1d32:61926 451                      31      0           216         216         86.9K       86.9K       .NET     0.0.1    4d0h12m59s  2016-06-14 14:18:35.4670198 -0400 EDT
  fe80::1ca1:378d:181f:1d32:61940 452                      0       0           0           1           0           343         .NET     0.0.1    4d0h12m15s  2016-06-10 14:06:55.9304056 -0400 EDT
  fe80::1ca1:378d:181f:1d32:55192 466                      0       0           0           6           0           1.3K        .NET     0.0.1    3h55m23s  2016-06-14 14:14:46.288869 -0400 EDT
  fe80::1ca1:378d:181f:1d32:55238 470                      0       0           0           0           0           0           .NET     0.0.1    3h51m19s  2016-06-14 10:27:37.689753 -0400 EDT
  fe80::1ca1:378d:181f:1d32:58159 489                      0       0           0           2.0K        0           273.0K      .NET     0.0.1    4m52s    2016-06-14 14:18:55.4706161 -0400 EDT
  fe80::1ca1:378d:181f:1d32:58160 490                      28      0           176         195         64.9K       64.9K       .NET     0.0.1    4m51s    2016-06-14 14:18:55.4695984 -0400 EDT
  10.133.161.7:8180    491                      1       0           1.9K        0           264.8K      0           .NET     0.0.1    4m21s    2016-06-14 14:18:55.4706161 -0400 EDT
```

**Comment by wallyqs** [comment]
Fixed via #39

**Comment by wallyqs** [comment]
Still occurs when not using reverse dns lookup :/

**Comment by wallyqs** [comment]
Closed via #41
